src/Goerli-DssSpellCollateral.sol

```diff
@@ -28,61 +77,280 @@ contract DssSpellCollateralAction {
+        CHANGELOG.setAddress("RWA008_A_OUTPUT_CONDUIT", RWA008_A_OUTPUT_CONDUIT);
+
+        REGISTRY.put(
+            "RWA008-A",
+            MCD_JOIN_RWA008_A,
+            RWA008,
+            GemJoinAbstract(MCD_JOIN_RWA008_A).dec(),
+            RWA008_REG_CLASS_RWA,
+            pip,
+            address(0),
+            "RWA008-A: SG Forge OFH",
+            GemAbstract(RWA008).symbol()
+        );
+    }
+
+    function onboardRwa009(
+        ChainlogAbstract CHANGELOG,
+        IlkRegistryAbstract REGISTRY,
+        address MIP21_LIQUIDATION_ORACLE,
+        address MCD_VAT,
+        address MCD_JUG,
+        address MCD_SPOT
+    ) internal {
+        // Set ilk bytes32 variable
+        bytes32 ilk = "RWA009-A";
+
+        // Sanity checks
+        require(GemJoinAbstract(MCD_JOIN_RWA009_A).vat() == MCD_VAT, "join-vat-not-match");
+        require(GemJoinAbstract(MCD_JOIN_RWA009_A).ilk() == ilk,     "join-ilk-not-match");
+        require(GemJoinAbstract(MCD_JOIN_RWA009_A).gem() == RWA009,  "join-gem-not-match");
+        require(
+            GemJoinAbstract(MCD_JOIN_RWA009_A).dec() == DSTokenAbstract(RWA009).decimals(),
+            "join-dec-not-match"
+        );
+
+        /*
+         * init the RwaLiquidationOracle
+         */
+        RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).init(ilk, RWA009_A_INITIAL_PRICE, RWA009_DOC, RWA009_A_TAU);
+        (, address pip, , ) = RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).ilks(ilk);
+
+        // Set price feed for RWA009
+        SpotAbstract(MCD_SPOT).file(ilk, "pip", pip);
+
+        // Init RWA009 in Vat
+        VatAbstract(MCD_VAT).init(ilk);
+        // Init RWA009 in Jug
+        JugAbstract(MCD_JUG).init(ilk);
+
+        // Allow RWA009 Join to modify Vat registry
+        VatAbstract(MCD_VAT).rely(MCD_JOIN_RWA009_A);
+
+        // Allow RwaLiquidationOracle2 to modify Vat registry
+        VatAbstract(MCD_VAT).rely(MIP21_LIQUIDATION_ORACLE);
+
+        // 100m debt ceiling
+        VatAbstract(MCD_VAT).file(ilk, "line", RWA009_A_INITIAL_DC);
+        VatAbstract(MCD_VAT).file("Line", VatAbstract(MCD_VAT).Line() + RWA009_A_INITIAL_DC);
+
+        // No dust
+        // VatAbstract(MCD_VAT).file(ilk, "dust", 0)
+
+        // 0% stability fee
+        JugAbstract(MCD_JUG).file(ilk, "duty", ZERO_PCT_RATE);
+
+        // collateralization ratio 100%
+        SpotAbstract(MCD_SPOT).file(ilk, "mat", RAY);
+
+        // poke the spotter to pull in a price
+        SpotAbstract(MCD_SPOT).poke(ilk);
+
+        // give the urn permissions on the join adapter
+        GemJoinAbstract(MCD_JOIN_RWA009_A).rely(RWA009_A_URN);
+
+        // DSS_PAUSE_PROXY permission on URN
+        RwaUrnLike(RWA009_A_URN).hope(address(this));
+
+        // lock RWA009 Token in the URN
+        DSTokenAbstract(RWA009).approve(RWA009_A_URN, 1 * WAD);
+        RwaUrnLike(RWA009_A_URN).lock(1 * WAD);
+
+        // draw DAI to genesis address
+        RwaUrnLike(RWA009_A_URN).draw(RWA009_DRAW_AMOUNT);
```

Comment:
I'd move this after finishing setting up the RWA to separate what is proper set up (and done just once) than what is minting and will happen more times.

Reply:
Maybe even consider taking it out of the "onboard" function.

Reply:
Moved this to the main spell file.